src/utils.coffee

```diff
@@ -271,7 +273,7 @@ exports.generate_transformation_string = (options) ->
   if options["offset"]?
     [options["start_offset"], options["end_offset"]] = split_range(utils.option_consume(options, "offset"))
 
-  overlay = process_layer(utils.option_consume(options, "overlay"))
+  overlay = process_layer(utils.option_consume(options, "overlay"), escapeQuotes)
```

Comment:
Shouldn't it be added to the underlay as well?

Reply:
@shedar , we should